src/pip/_internal/commands/install.py

```diff
@@ -434,7 +434,7 @@ def run(self, options, args):
             message = create_env_error_message(
                 error, show_traceback, options.use_user_site,
             )
-            logger.error(message, exc_info=show_traceback)
+            logger.exception(message)
```

Comment:
This is a behaviour change.

Reply:
Sorry for my carelessness, I'm reverting it now.

Reply:
Could you try `logger.exception(message, exc_info=show_traceback)` instead? I am not sure if even that will count as a behaviour change, but I think you can try both of them when hitting this error condition, and see if the output from pip changes.

Reply:
That'd be complained by G202: Logging statements should not use redundant `exc_info=True` in `exception`.  The problem here is that `show_traceback` is evaluated as constant `True` somehow and I haven't investigate any further (nor want to do it until we decide if we want to use it).